specs/test_formats/sanity/slots.md

```diff
@@ -4,14 +4,34 @@ Sanity tests to cover a series of one or more empty-slot transitions being proce
 
 ## Test case format
 
+### `meta.yaml`
+
 ```yaml
-description: string    -- description of test case, purely for debugging purposes
+description: string    -- Optional. Description of test case, purely for debugging purposes.
 bls_setting: int       -- see general test-format spec.
-pre: BeaconState       -- state before running through the transitions.
-slots: N               -- amount of slots to process, N being a positive number.
-post: BeaconState      -- state after applying all the transitions.
 ```
 
+
+### `pre.yaml`
+
+A YAML-encoded `BeaconState`, the state before running the transitions.
+
+A `pre.ssz` is also available as substitute.
+
+
+### `slots.yaml`
+
+An integer. The amount of slots to process (i.e. the difference in slots between pre and post), always a positive number.
```

Comment:
include `.ssz` substitute?

Reply:
The tool currently only encodes bare byte arrays as ssz, and ssz objects. Not sure if a slots number counts as such, but will add it for consistency.

Reply:
Nevermind, it does feel hacky from a test-generation perspective to provide such integer as a different format. And I don't expect the ssz file to be read. May reconsider later.